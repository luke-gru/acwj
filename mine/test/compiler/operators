int main() {
  int a, b, x;
  a = b = 10;
  x = a + b;
  if (x) { printstring("x is not zero\n"); }
  if (!x) { printstring("x is zero\n"); }
  x = 0;
  if (x) { printstring("x is not zero\n"); }

  a = 1;
  printint(a++);
  printint(a);
  printint(++a);
  printint(a);

  return(0);
}
