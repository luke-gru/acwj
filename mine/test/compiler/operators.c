int main() {
  int a;
  int b;
  int x;
  a = b = 10;
  x = a + b;
  if (x) { printstring("x is not zero\n"); }
  if (!x) { printstring("x is zero\n"); }
  x = 0;
  if (x) { printstring("x is not zero\n"); }

  a = 1;
  printint(a++);
  printint(a);
  printint(++a);
  printint(a);
  printint(10%3);

  return(0);
}
