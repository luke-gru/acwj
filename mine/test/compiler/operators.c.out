x is not zero
1
2
3
3
